src/game_constants.h

```diff
@@ -1,6 +1,9 @@
 #ifndef GAME_CONSTANTS_H
 #define GAME_CONSTANTS_H
 
+/** version number for unique item identifiers @see game::next_item_uid */
+#define ITEM_UID_VERSION 0
```

Comment:
Is there any reason why this is global and public? What does any code outside of the uid-generator function care about this? It's only used at one point, it should be declared there as `constexpr`.